modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -9,6 +9,9 @@
 [role="_abstract"]
 The `wasp-agent` component facilitates memory overcommitment by assigning swap resources to worker nodes. It also manages pod evictions when nodes are at risk due to high swap I/O traffic or high utilization.
 
+The `wasp-agent` component is deployed automatically if `memoryOvercommitPercentage` is set to more than `100` when you first create the `HyperConverged` custom resource (CR).
+
+
```

Comment:
since the "wasp-agent" component now deployeh by HCO - I guess we need to remove some steps from the installation part.

Can't add inline comments, so will add it here.

In the doc we need to keep only:
```
1. Configure the kubelet service to permit swap usage
2. Provision swap by creating a MachineConfig object.

9. Enable memory overcommitment in OpenShift Virtualization by using the web console or the CLI.
```

these lines can be removed:
https://github.com/openshift/openshift-docs/pull/102156/files#diff-6c4e57708f8c064606b3df0f6729426f324069abe615aeddf390c84d32b2b92dR131-R281


in verification part - the command should be updated:
```
$ oc rollout status ds wasp-agent -n openshift-cnv
```